alembic/latiss/versions/535c454d7311_add_quicklook.py

```diff
@@ -579,19 +579,10 @@ def upgrade() -> None:
     )
 
     # Need to drop and re-create visit1 view before dropping columns
-    op.execute("DROP VIEW IF EXISTS cdb_latiss.visit1")
```

Comment:
Hm. Rather than changing history, I'd rather that the changes apply only to future migrations.